contracts/SaveDAI.sol

```diff
@@ -63,24 +62,24 @@ contract SaveDAI is ISaveDAI, ERC20, ERC20Detailed, Pausable, Ownable {
         ocDaiExchange = _getExchange(ocDaiAddress);
 
         require(
-            dai.approve(address(this), LARGE_APPROVAL_NUMBER) && 
+            dai.approve(address(this), LARGE_APPROVAL_NUMBER) &&
```

Comment:
Isn't this just giving ourselves approval to spend our own dai?

Reply:
@dougiebuckets @slgraham Good point. I removed this, along with a couple other `approve` and `transferFrom` and replaced with `transfer` (when transferring from the saveDAI contract). Rather then approving and transferFrom, we can just directly transfer from our contract.

Reply:
@slgraham good catch!

@kseniya292 - Lolo can't believe we missed that until now :)